sophia-vote-contract.md

```diff
@@ -100,37 +106,37 @@ contract Vote =
    record state = {
       votes: map(address, candidates) }
 
-   public stateful function init() : state = {
+   stateful entrypoint init() : state = {
```

Comment:
This function does nothing `stateful` so the keyword can be removed.

Reply:
I have implemented the requested changes. Can you resolve the conversation?